src/res/KM_Resource.pas

```diff
@@ -69,6 +69,8 @@   TKMResource = class
     procedure ExportTreeAnim;
     procedure ExportHouseAnim;
     procedure ExportUnitAnim;
+
+    function GetMaxPrintWidthOfStrings(aStrings: array of string; aFont: TKMFont): Integer;
```

Comment:
This should be in TKMResFonts, since it uses only Fonts info. Somewhere next to "TKMFontData.GetTextSize"